lib/src/mixins/dartdoc.dart

```diff
@@ -6,10 +6,10 @@ import 'package:built_collection/built_collection.dart';
 
 abstract class HasDartDocs {
```

Comment:
Let's add comments here and below why `Object` is being used and what a valid type is (and isn't).

Reply:
Done